app/utils.py

```
@@ -0,0 +1,24 @@
+import json
+import math
+from typing import Dict, Any, List
+
+
+Location = List[float, int]
+
+
+def load_config(file_path: str) -> Dict[str, Any]:
+    try:
+        with open(file_path, "r") as file:
+            read_config = json.load(file)
+        return read_config
+    except FileNotFoundError:
+        print(f"Error: Configuration file not found at {file_path}")
```

In the except block you print an error but do not return or raise. That causes `load_config` to return `None` on failure, which will break callers expecting a dict (config). The spec requires loading `config.json` successfully (or failing loudly) per: "1.2. Load configuration from a JSON file named config.json that provides: - FUEL_PRICE ... (Ensure the path to the JSON file is correct; see checklist item 4)". Either re-raise the FileNotFoundError (or a custom exception) or return a clear sentinel and document it. Also consider handling `json.JSONDecodeError` to report malformed JSON.